android/src/main/java/com/instabug/instabugflutter/InstabugFlutterPlugin.java

```diff
@@ -997,4 +997,16 @@ public void disable() {
         Instabug.disable();
     }
 
+    /**
+     * Sets tracking of usersteps.
+     *
+     * @param isEnabled isEnabled flag
+     */
+    public void setTrackUserSteps(boolean isEnabled) {
+        if isEnabled {
```

Comment:
Java needs brackets here 

```suggestion
        if (isEnabled) {
```